freqtrade/exchange/exchange.py

```diff
@@ -2055,6 +2055,42 @@ def _set_leverage(
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
+    def get_interest_rate(self) -> float:
+        """
+        Calculate interest rate - necessary for Margin trading.
+        """
+        return 0.0
+
+    def get_liquidation_price(
```

Comment:
I moved (and renamed and simplified) this method to be in the exchange class (which i think is where it belongs).
there's little point in having it return a Tuple if the 2nd value has no meaning. 

It complicates usage beyond reason - especially considering that the interest-rate part is not done (and may even require different arguments once we get to that).